tests/zkevm/test_worst_compute.py

```diff
@@ -237,23 +243,173 @@ def test_worst_modexp(
 
 
 @pytest.mark.valid_from("Cancun")
-def test_worst_ecrecover(
+@pytest.mark.parametrize(
+    "precompile_address,parameters",
+    [
+        pytest.param(
+            0x01,
+            [
+                "38D18ACB67D25C8BB9942764B62F18E17054F66A817BD4295423ADF9ED98873E",
```

Comment:
Please add one line comment for each of these hex inputs where are they from and why they are worst cases.

Reply:
This is a regular signature. Do you already know some quirks for other input be a better candidates? We can further iterate on it.

I'll add a comment explaining this is a regular signature and add a TODO if we can find a better input.